File: AI/robots/robot_2022.js
'use strict';

const Robot = require('./robot')

module.exports = class Robot2022 extends Robot{
    constructor(app){
        super(app);
    }

    triggerLightHouse(parameters){
        this.app.logger.log("--> Triggering LightHouse"+JSON.stringify(parameters));
        return true;
    }
}

File: pythonAI/robots/robot_2022.py
from robots.robot import Robot

class Robot2022(Robot):

    def __init__(self, app):
        super().__init__(app)
    

    def triggerLightHouse(self, parameters):
        self.app['logger'].info("--> Triggering LightHouse " + str(parameters))
        return True

    
